mesonbuild/interpreter/primitives/integer.py

```diff
@@ -53,6 +48,14 @@ def __init__(self, obj: int, interpreter: 'Interpreter') -> None:
     def display_name(self) -> str:
         return 'int'
 
+    def operator_call(self, operator: MesonOperator, other: TYPE_var) -> TYPE_var:
+        if isinstance(other, bool):
+            #raise InvalidArguments(f'The {operator.value} of int does not accept objects of type bool ({other})')
```

Comment:
What is the purpose of the comment

Reply:
... optimism for meson 2.0? :D

Reply:
Haha. I wish! Wanna add a space after the octothorpe and maybe add some context that the comment should be kept around for 2.0?